swap/swap.go

```diff
@@ -273,13 +287,13 @@ func (s *Swap) handleEmitChequeMsg(ctx context.Context, p *Peer, msg *EmitCheque
 	defer p.lock.Unlock()
 
 	cheque := msg.Cheque
-	auditLog.Info("received cheque from peer", "peer", p.ID().String(), "honey", cheque.Honey)
+	p.logger.Info("received cheque from peer", "honey", cheque.Honey)
 	_, err := s.processAndVerifyCheque(cheque, p)
 	if err != nil {
 		return err
 	}
 
-	auditLog.Debug("received cheque processed and verified", "peer", p.ID().String())
+	p.logger.Debug("received cheque, processed and verified")
```

Comment:
wouldn't it be good to log which cheque was processed and verified? without any identifying details this logline doesn't say much

Reply:
we could add `honey` as in line `307`, but other than that, i'm not sure which field could identify a cheque in a way that is useful for logs.

tagging @ralph-pichler for insight

Reply:
The `CumulativePayout` and `Contract` fields should be enough to uniquely identify which cheque it is. Instead of `Contract` `p.beneficiary` might also be useful (this would be the signer of the cheque here).

Reply:
added `Beneficiary` and `CumulativePayout` to this `Debug` log entry

thanks guys